docker-entrypoint.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+set -e
+
+SUPERVISOR_APP_NAME=$1
+APP_WSGI=$2
+APP_PORT_NUMBER=$3
+DJANGO_SETTINGS_MODULE="${APP_WSGI:0:${#APP_WSGI}-5}.settings"
```

Comment:
Why do these things need to be configurable? They're all set in the dockerfile's `ENTRYPOINT`, which means they can't be overridden.